### DISCUSSION TITLE
Installed PostGIS and created spatial_ref_sys table in public schema. Now need to enable RLS for this table.

### DISCUSSION THREAD

**Discussion by acanturk1** [supabase/supabase#19143] [Questions] [answered]
Hello,

I Installed PostGIS extension using Subabase portal.

I have been asked which schema do I want to install the extension. I selected public. Not sure if this is the best approach.

Now it has created a spatial_ref_sys table in public schema which I believe need to enable RLS for this table. But I get this error:

`Failed to toggle RLS: failed to update pg.tables with the given ID: must be owner of table spatial_ref_sys`

Is it OK to install it on public schema? If yes, how can I secure my table?

Thank you.

**Comment by steve-chavez** [comment, marked as answer]
Do not create extensions on the public schema (this will make the internal extension tables available through the REST/GraphQL APIs). The UI has an extensions page where it will automatically handle this for you.

If you wish to create the extension with SQL, do:

```sql
CREATE EXTENSION postgis SCHEMA extensions;
```
